GVFS/GVFS.Common/Maintenance/GitMaintenanceStep.cs

```diff
@@ -189,29 +199,25 @@ protected void SaveLastRunTimeToFile()
             }
         }
 
-        protected void LogErrorAndRewriteMultiPackIndex(ITracer activity, GitProcess.Result result)
+        protected void LogErrorAndRewriteMultiPackIndex(ITracer activity)
         {
             EventMetadata errorMetadata = this.CreateEventMetadata();
-            errorMetadata["MultiPackIndexVerifyOutput"] = result.Output;
-            errorMetadata["MultiPackIndexVerifyErrors"] = result.Errors;
             string multiPackIndexPath = Path.Combine(this.Context.Enlistment.GitPackRoot, "multi-pack-index");
             errorMetadata["TryDeleteFileResult"] = this.Context.FileSystem.TryDeleteFile(multiPackIndexPath);
 
-            GitProcess.Result rewriteResult = this.RunGitCommand((process) => process.WriteMultiPackIndex(this.Context.Enlistment.GitObjectsRoot));
+            GitProcess.Result rewriteResult = this.RunGitCommand((process) => process.WriteMultiPackIndex(this.Context.Enlistment.GitObjectsRoot), nameof(GitProcess.WriteMultiPackIndex));
             errorMetadata["RewriteResultExitCode"] = rewriteResult.ExitCode;
 
             activity.RelatedError(errorMetadata, "multi-pack-index is corrupt after write. Deleting and rewriting.");
         }
 
-        protected void LogErrorAndRewriteCommitGraph(ITracer activity, GitProcess.Result result, List<string> packs)
+        protected void LogErrorAndRewriteCommitGraph(ITracer activity, List<string> packs)
         {
             EventMetadata errorMetadata = this.CreateEventMetadata();
-            errorMetadata["CommitGraphVerifyOutput"] = result.Output;
-            errorMetadata["CommitGraphVerifyErrors"] = result.Errors;

```

Comment:
We are no longer capturing the `Output`?  Are there cases where we want that in the tracing?

Reply:
For consistency we're only printing errrors for maitence jobs.  We should be able to look it up any error/output from git using Trace2 if necessary.  So yes not needed.

Reply:
Would we want the errors to be in the logs written out to file? We might not always have access to the trace2 data, and I could see value in being able to look up the error in the local log file. Do we trace enough other data to some information about the error in the local log file?

Reply:
@jamill we don't log the output for other git commands, and I really don't think it's a necessity.   The errors will be in the log.   If we need to look up more info we can in Trace2.  

I've done a lot of error looksups myself already and I find Trace2 to have vastly more info and be super useful.  

Of course we can change our mind on this in the future.  But so far with my investigations this is my recommended way to go.

Reply:
>We will do it in a central spot now in RunGitProcess so we can limit message size.

Ah! I see that now.